src/schema/schema.ts

```diff
@@ -20,6 +20,7 @@ import {legend} from './fielddef.legend.schema';
 // TODO(#620) refer to vega schema
 // var vgStackSchema = require('vega/src/transforms/Stack').schema;
 
+export {aggregate} from './fielddef.schema';
```

Comment:
better create `vl.aggregate` in `aggregate.ts` and
- move  `AGGREGATE_OPS`  to `vl.aggregate.OPS`
- move `supportedEnums` and `supportedTypes` to `vl.aggregate`